routes.yaml

```diff
@@ -2,6 +2,5 @@ app_name: company-appointments.api.ch.gov.uk
 group: api
 weight: 900
 routes:
-  1: ^/company/.*?/appointments(/.*?|$)$
-  2: ^/company/(.){0,10}/officers$
-  3: ^/officers/.*/appointments$
\ No newline at end of file
+  1: ^/company/.*?/appointments(/.*?|$)/full_record(/.*?|$)$
+  2: ^/company/.*?/appointments$
```

Comment:
This route does not need exposed in live until we are ready to release the sub deltas and so should be removed.